cmd/cloud/workspace.go

```diff
@@ -457,7 +457,7 @@ func newRemoveOrganizationTokenWorkspaceRole(out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "remove [ORG_TOKEN_ID]",
 		Short: "Remove an Workspace API token's Deployment Role",
-		Long:  "Remove an Workspace API token's Deployment Role\n$astro workspace token organization-token remove [ORG_TOKEN_ID] --name [token name].",
+		Long:  "Remove an Workspace API token's Deployment Role\n$astro workspace token organization-token remove [ORG_TOKEN_ID] --org-token-name [token name].",
```

Comment:
Is this correct? The function name is different from what we are calling.

Reply:
it was not good eyes. Should be good now.